docs/officers/dues/data-model.rst

```diff
@@ -0,0 +1,49 @@
+Data Model
+==========
+
+Dues are tracked in Apiary using several related objects.
+
+-----------
+Fiscal Year
+-----------
+
+A **Fiscal Year** groups together several `Dues Package`_ and `Merchandise`_ objects.
+Fiscal years are primarily used for reporting and do not affect any functionality.
+They need to be created manually once a year to group the objects for that year.
+
+---------------
+_`Dues Package`
+---------------
+
+A **Dues Package** represents an option for paying dues.
+In the member-facing interface, packages are labeled as "Dues Terms."
+There are a few key attributes:
+
+- The **cost** of the package
+- The **membership start date**, when someone is considered to be an active member if they purchase this package
+- The **membership end date**, when someone is no longer considered to be an active member unless/until they purchase another package
+- The **access start date**, when someone will gain access to RoboJackets systems if they purchase this package
+- The **access end date**, when someone will lose access to RoboJackets systems unless/until they purchase another package
+
+Additional attributes are used to determine whether a package may be offered to a specific person, based on their affiliation with Georgia Tech and prior purchase history.
```

Comment:
```suggestion
Additional attributes are used to determine whether a package may be offered to a specific person, based on their affiliation with Georgia Tech and prior purchase history.

Generally it's a good idea to look at past fiscal years for examples for these settings.
```

Reply:
The defaults should be fine; I'm not sure I want to encourage changing any of them.

Reply:
I'd forgotten about the action that creates everything.